master/internal/rm/kubernetesrm/kubernetes_resource_manager.go

```diff
@@ -638,7 +637,7 @@ func (k *ResourceManager) createResourcePoolSummary(
 		SlotType:                     k.config.SlotType.Proto(),
 		DefaultAuxPool:               k.config.DefaultAuxResourcePool == poolName,
 		DefaultComputePool:           k.config.DefaultComputeResourcePool == poolName,
-		Preemptible:                  preemptible,
+		Preemptible:                  false,
```

Comment:
Why not remove `Preemptible` completely?

Reply:
the dispatcher/agent/k8s resource managers all use type `resourcepoolv1.ResourcePool`, and dispatcher RM sets this value to true (and now agent & k8s are always false)

Reply:
Makes sense, thanks for explaining!